src/types.ts

```diff
@@ -369,6 +369,22 @@ export interface Price {
     price: BigNumber;
 }
 
+interface BasePriceResponse {
+    price: BigNumber;
+    buyAmount: BigNumber;
+    sellAmount: BigNumber;
+    sellTokenAddress: string;
```

Comment:
This uses the token address to be in line with the quote endpoint. However, the prices endpoint returns symbols.

Either symbol or token address would work as long as it's consistent.

Reply:
@kimpers are you referring to https://api.0x.org/swap/v0/prices ?

Reply:
Yeah exactly.

It would be nice to have consistency across the board, if we go with token address here maybe we should add it there as well or vice versa.

Reply:
I agree it should be consistently address since we cannot guarantee symbol.